api.go

```diff
@@ -39,9 +39,13 @@ type FileMatch struct {
 
 	// Repository is the globally unique name of the repo of the
 	// match
-	Repository  string
-	Branches    []string
-	LineMatches []LineMatch
+	Repository string
+	Branches   []string
+
+	// One of LineMatches or ChunkMatches will be returned depending on whether
+	// the SearchOptions.ChunkMatches is set.
+	LineMatches  []LineMatch
```

Comment:
I thought you removed LineMatches in this PR?

Reply:
hah, this is what happens when i quickly browse over a PR. I totally missed you generate the line matches from the chunks. nice!